sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFieldIdIOSuite.scala

```diff
@@ -107,6 +107,35 @@ class ParquetFieldIdIOSuite extends QueryTest with ParquetTest with SharedSparkS
     }
   }
 
+  test("absence of field ids: reading nested schema") {
```

Comment:
Although it's not a mandatory in this case, if you don't mind, please add `SPARK-38094: `.
```scala
- test("absence of field ids: reading nested schema") {
+ test("SPARK-38094: absence of field ids: reading nested schema") {
```